lib/ExpensiMark.js

```diff
@@ -214,6 +214,14 @@ export default class ExpensiMark {
                 regex: /<(del)(?:"[^"]*"|'[^']*'|[^'">])*>(.*?)<\/\1>(?![^<]*(<\/pre>|<\/code>))/gi,
                 replacement: '~$2~'
             },
+            {
+                name: 'quote',
+                regex: /\n?<(blockquote|q)(?:"[^"]*"|'[^']*'|[^'">])*>([\s\S]*?)<\/\1>(?![^<]*(<\/pre>|<\/code>))/gi,
+                replacement: (match, g1, g2) => {
+                    const resultString = g2.trim().split('\n').map(m => `> ${m || ''}`).join('\n');
```

Comment:
Also can we safely simplify this like so:

```suggestion
                    const resultString = g2.trim().split('\n').map(m => `> ${m}`).join('\n');
```

Reply:
Asking earnestly, but I don't think `.split` would give us falsey array elements, right?

Reply:
Correct.